src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -182,9 +189,9 @@ protected TokenValidationParameters(TokenValidationParameters other)
             NameClaimType = other.NameClaimType;
             NameClaimTypeRetriever = other.NameClaimTypeRetriever;
             PropertyBag = other.PropertyBag;
-            RequireAudience = other.RequireAudience;
-            RequireExpirationTime = other.RequireExpirationTime;
-            RequireSignedTokens = other.RequireSignedTokens;
+            _requireAudience = other._requireAudience;
+            _requireExpirationTime = other._requireExpirationTime;
```

Comment:
here and elsewhere, no need to change getter usage, will keep old format for consistency.